node_modules/oae-core/resetpassword/resetpassword.html

```diff
@@ -0,0 +1,30 @@
+<!-- CSS -->
+<link rel="stylesheet" type="text/css" href="css/resetpassword.css" />
+
+<!-- MODAL -->
+<div id="resetpassword-modal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="resetpassword-modal-title" aria-hidden="true">
+    <div class="modal-dialog modal-sm">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&#215;</button>
+                <h3 id="resetpassword-modal-title">__MSG__RESET_PASSWORD__</h3>
+            </div>
+			<div class="modal-body">
+		        <form id="resetpassword-input-username" class="form-horizontal" role="form">
+                    <div class="form-group">
+                        <label class="control-label col-lg-5" for="resetpassword-new-password">__MSG__USERNAME_COLON__</label>
+                        <div class="col-lg-7">
+                            <input type="text" id="resetpassword-username" name="username" placeholder="username" class="form-control required maxlength-short">
+                        </div>
+                    </div>
+		            <div class="text-center">
+		                <button type="submit" class="btn btn-primary">Submit</button>
```

Comment:
'Submit' should go to a bundle file as well - probably into `/shared/oae/bundles/ui/` as it's likely it'll get used elsewhere as well.